packages/components/src/Typography/css/FontWeights.css

```diff
@@ -3,6 +3,11 @@
   font-weight: 400;
 }
 
+.semiBold {
```

Comment:
Adding this to match the Figma.  `semiBold` matches the naming for `600` font-weight in mobile